dora/core/server/master/src/main/java/alluxio/master/job/UfsFileIterable.java

```diff
@@ -140,7 +141,7 @@ private FileInfo transformUfsStatus(UfsStatus ufsStatus) {
       AlluxioURI ufsUri = new AlluxioURI(PathUtils.concatPath(mPath,
           CommonUtils.stripPrefixIfPresent(ufsStatus.getName(), mPath)));
       FileInfo info = new FileInfo().setName(ufsUri.getName()).setPath(ufsUri.getPath())
-                                    .setUfsPath(ufsUri.getPath())
+                                    .setUfsPath(ufsUri.toString())
```

Comment:
whats the diff between getpath and tostring?

Reply:
toString give you full ufs path with schema&authority.